### PR TITLE
Update AcknowledgeBulkWriteResult.java

### PR DESCRIPTION
**Pull request by MorrisLaw**
Fixed the exception error in getModifiedCount and removed the word "this" from "the server this must".... also added version 2.6 "or higher" to the end of the statement. Also added spacing to the toString return values.

### CODE REVIEW

**Comment by Jeremy1229** [comment]
"is" not "this" was the grammatical error I found in the exception error under getModifiedCount.

File: driver/src/main/com/mongodb/AcknowledgedBulkWriteResult.java [hunk 1]
```diff
@@ -120,11 +120,11 @@ public int hashCode() {
     @Override
     public String toString() {
         return "AcknowledgedBulkWriteResult{"
-               + "insertedCount=" + insertedCount
-               + ", matchedCount=" + matchedCount
-               + ", removedCount=" + removedCount
-               + ", modifiedCount=" + modifiedCount
-               + ", upserts=" + upserts
+               + "insertedCount = " + insertedCount
```
Comment (jyemin) [inline driver/src/main/com/mongodb/AcknowledgedBulkWriteResult.java]: Please revert this change.  We've been using IntelliJ's default toString generator and this will introduce an inconsistency with all the other generated methods.

**Comment by rozza** [comment]
Thanks @MorrisLaw  - I updated the exception based on your PR, but left the `toString()` method as is.
